commit 75f56b80c83ecac504f30a38c2304fbb8194c617 (refs/changes/47/5247/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-12-14T14:53:42-08:00 (10 months ago)
    
    TRITON-1045 mockcloud machine_destroy needs to use uuid instead of req.uuid

diff --git a/lib/backends/dummy/tasks/machine_destroy.js b/lib/backends/dummy/tasks/machine_destroy.js
index 60a7c53..fdf3c07 100644
--- a/lib/backends/dummy/tasks/machine_destroy.js
+++ b/lib/backends/dummy/tasks/machine_destroy.js
@@ -45,7 +45,7 @@ function start() {
     vasync.pipeline({
         funcs: [
             function ensureProvisionComplete(_, cb) {
-                common.ensureProvisionComplete(self.req.uuid, cb);
+                common.ensureProvisionComplete(uuid, cb);
             },
             function deleteVm(_, cb) {
                 /* this will pass the error (if any) to _pipelineCompleted */
diff --git a/package.json b/package.json
index 55b3579..f5d79ca 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.6.0",
+  "version": "2.6.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
